lottie/src/main/java/com/airbnb/lottie/CompositionLayer.java

```diff
@@ -106,4 +108,16 @@ boolean hasMatte() {
     }
     return false;
   }
+
+  @Override
+  public void addColorFilter(@Nullable String layerName, @Nullable String contentName,
```

Comment:
ok i did understand your original point, but I think the current code supports that already. As long as the layerName is the precomp name.